pkg/naming/constants.go

```diff
@@ -40,13 +40,23 @@ const (
 	ScyllaServiceTypeMember   ScyllaServiceType = "member"
 )
 
+type ScyllaIngressType string
+
+const (
+	ScyllaIngressTypeDirect ScyllaIngressType = "direct"
+	ScyllaIngressTypeAny    ScyllaIngressType = "any"
+
+	ScyllaIngressAnySubdomain = "any"
```

Comment:
this may not need a constant e2e should inline it to enforce explicit changes to surface the impact